### ISSUE TITLE
App crashes when typing in filter field in event view

### ISSUE THREAD

**Issue by rgauthey** [aptakube/aptakube#312]
### Current and expected behavior

When typing any text into the filter input in the event view, the app crashes with the following error:

```typescript
TypeError: undefined is not an object (evaluating 'f[T].visible')

@tauri://localhost/assets/index-C4bMzrKU.js:1013:3013
filter@[native code]
@tauri://localhost/assets/index-C4bMzrKU.js:1013:2999
oK@tauri://localhost/assets/index-C4bMzrKU.js:1:5909
Ym@tauri://localhost/assets/index-C4bMzrKU.js:1:5836
_C@tauri://localhost/assets/index-C4bMzrKU.js:1:6650
g9@tauri://localhost/assets/index-C4bMzrKU.js:1:7002
aK@tauri://localhost/assets/index-C4bMzrKU.js:1:6889
fl@tauri://localhost/assets/index-C4bMzrKU.js:1:6821
f9@tauri://localhost/assets/index-C4bMzrKU.js:1:5542
```

The issue was reproduced on both a k3s and GKE cluster.

### Environment & Operating System

Aptakube version: 1.7.12 (20241014.092245)
OS: MacOS 15.0.1 (24A348)
GKE : v1.30.5-gke.1014001
k3s : v1.30.4+k3s1

**Comment by goenning** [comment]
Thanks for sharing that, I’ll have a look soon.

if you clear the cache on the app settings > help > clear cache, does it still crash the app?

**Comment by rgauthey** [comment]
Looks like clearing the cache fixed it !

Thanks

**Comment by goenning** [comment]
Fixed for good (I hope!) on 1.7.13 :)
